data/json/snippets/effect_on_conditions.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_1",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  It is quite dense and appears to be several regular polyhedrons next to each other.",
+      "You see what seems to be a new constellation in the sky.  It appears to be a spiral with the central star constantly shifting color.",
+      "You see several shooting stars.  They seem larger than normal and are all a deep purple.",
+      "You see a shooting star.  It is bright red and does not fall in a straight line.",
+      "For a brief moment you see a shape in the sky roughly the size of the moon, but square and green.",
+      "For a brief moment, the stars and moon all disappear.  When they come back, you're not sure if they're in the same places as they were."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_2",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  It changes shape as you look from a triangle to a pentagon.",
+      "You see several shooting stars.  After they go below the horizon a bright glow briefly appears.",
+      "You see a shooting star.  It is bright red and stops falling after a second to hover there ominously.",
+      "For a brief moment you see a shape in the sky roughly the size of the moon.  It almost looks like like an eye blinking.",
```

Comment:
Good spotting! (I'd say "looked like an", BTW.)

Reply:
Present followed by past time, my brain tells me to keep the present but I'm not fluent enough to assert the valid conjugation. ^^"

Reply:
"For a brief moment you see" - so you saw for just a short period of time, and - to me at least - now you're thinking back on it. I can see (no pun intended!) arguments otherwise, including both being present tense and both being past tense.

Reply:
I agree. The character caught something for a fleeting moment, but it's gone now. It no longer *looks* like anything, but it *had looked* like something before. In fact, I might suggest the use of the past perfect instead of the past simple.

```suggestion
      "For a brief moment you see a shape in the sky roughly the size of the moon.  It had almost looked like a blinking eye.",
```